src/aura/PostContentTab/PostContentTabController.js

```diff
@@ -0,0 +1,15 @@
+({
+	onRender: function (component, event, helper) {
+		if (component.get('v.simpleRecord') != undefined)
```

Comment:
Using strict equal and strict not equal instead of the simple versions helps to maintain data type integrity through the code.